docs/index.md

```diff
@@ -6,22 +6,26 @@ sidebar:
 ---
 
 # Overview
-Tardigrade is a multi-cloud account provisioning and maintenance framework built with Terragrunt and Terraform. Currently, this implementation demonstrates the ability to manage AWS accounts, but can easily be extended to manage accounts on other cloud platforms.
+Tardigrade is a multi-cloud account provisioning and maintenance framework built
+with Terragrunt and Terraform. Currently, this implementation demonstrates the
+ability to manage AWS accounts, but can easily be extended to manage accounts on
+other cloud platforms.
 
 # Prerequisites
 * [terraform >= 0.12](https://www.terraform.io/)
   - credentials configured for an AWS account
 * [terragrunt >= v0.21](https://github.com/gruntwork-io/terragrunt)
 
 # How To Use
-You can use the following instructions to see the baseline infrastructure deployed to your account
+You can use the following command to see the baseline infrastructure deployed to
+your account:
 
 ```
 terragrunt plan --terragrunt-working-dir tardigrade/aws/609421024627/base --terragrunt-source-update
```

Comment:
Should the `tardigrade/aws/609421024627/base` be changed to `tardigrade/aws/{account_id}/member`?

Reply:
yep, fixed.